dao/src/test/java/cz/idomatojde/TestObjects.java

```diff
@@ -0,0 +1,41 @@
+package cz.idomatojde;
+
+import cz.idomatojde.entity.Category;
+import cz.idomatojde.entity.Offer;
+import cz.idomatojde.entity.User;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+public class TestObjects {
```

Comment:
TestObjectFactory maybe? To represent functionality

Reply:
"In Factory pattern, we create object without exposing the creation logic to the client and refer to newly created object using a common interface."
I think of these as "constant" fields in a static class. But classes in Java cannot be static ;)

Reply:
I was just referring to the name of the class